src/ContMechTensors.jl

```diff
@@ -315,7 +352,7 @@ for TensorType in (SymmetricTensor, Tensor)
             exp = tensor_create(get_type(Tt),f)
             return quote
                 $(Expr(:meta, :inline))
-                $($TensorType){order, dim, T, $N}($exp)
```

Comment:
Can the `N` in this function a few lines up be removed? It doesn't seem to be used. Same for other functions.